src/views/templates/template-details/TemplateDetailsView.scss

```diff
@@ -0,0 +1,123 @@
+@import 'styles/mixins';
+@import 'styles/palette';
+
+$padding: 20px 0;
+
+
+/* layout */
+.root {
+  @include centered;
+  height: 100%;
+  .container {
+    @include colCenter;
+    padding: $padding;
+    width: 50%;
+  }
+}
+
+section {
```

Comment:
This is ultra risky. You making all section markups in app styled like that. Put all code under .container. CSS modules don't use shadow DOM like in Angular. They creating unique class names. Use class instead if you want style it like that